.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
```

Comment:
- The use of sed commands to manipulate the RELEASE_NOTES variable can be error-prone and difficult to maintain. Consider using a more structured approach or a dedicated library for handling markdown, HTML, and other transformations.